### DISCUSSION TITLE
Worth forking now or should I wait?

### DISCUSSION THREAD

**Discussion by TomKranenburg** [DanRuta/xVA-Synth#41] [General]
How are you getting along with v3.0, @DanRuta? I really want to contribute to this project as I think it's amazing. Thought I might wait until the new release to start poking through the code though.

Been following you on patreon as well. So I think you're pretty close to release?

**Comment by DanRuta** [comment]
Hey! I've been committing code changes to the repo throughout the development, so what's in the repo now is the most up-to-date v3 code. v3 is currently in beta breaking/testing stage, so aside from bug fixes, this is likely it for now (for v3.0.0), in terms of newly added features

**Comment by TomKranenburg** [comment]
Cool. Give me a moment I'll help you. I'll start poking it with a stick.

**Comment by DanRuta** [comment]
I think the python deps .txt is up-to-date (I'll check), but I think there were a couple of library tweaks/fixes I had to make for things to work correctly. I'll have a look to see if I can single them out, but if you do run into any weird python issues out-of-the-box, or dependencies-related issues, let me know. You can run the frontend+backend with `npm start`. I'll get some more documentation going soon

**Comment by TomKranenburg** [comment]
Awesome. Tell you what I'll get it set up then write out some build instructions to help you out a bit. I've been using your system for a while it's the least I can do.

**Comment by DanRuta** [comment]
Ah awesome, I appreciate it! It would be cool to bring on some more community development to xVASynth/xVATrainer, for sure!
